envi/archs/arm/disasm.py

```diff
@@ -1083,13 +1083,20 @@ def p_coproc_load(opval, va):
         iflags = IF_L
     else:
         iflags = 0
-
-    olist = (
-        ArmCoprocOper(cp_num),
-        ArmCoprocRegOper(CRd),
-        ArmImmOffsetOper(Rn, offset*4, va, pubwl=punwl),
-    )
-    
+    #check for index. Non-index is option
```

Comment:
you keep adding commits to this branch before i can merge it, which is making it more and more difficult to make sure everything is appropriately handled.  this latest merge should have been a new branch from atlas_arm_fixes.